modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/lar/PortletDataHandlerRegistryImpl.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.exportimport.internal.lar;
+
+import com.liferay.exportimport.kernel.lar.PortletDataHandler;
+import com.liferay.exportimport.kernel.lar.PortletDataHandlerRegistry;
+import com.liferay.osgi.service.tracker.collections.map.ScopedServiceTrackerMap;
+import com.liferay.osgi.service.tracker.collections.map.ScopedServiceTrackerMapFactory;
+
+import org.osgi.framework.BundleContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+
+/**
+ * @author Alejandro Tardín
+ */
+@Component(service = PortletDataHandlerRegistry.class)
+public class PortletDataHandlerRegistryImpl
```

Comment:
PortletDataHandlerRegistryImpl and PortletDataHandlerRegistry should be inlined into PortletBagImpl. After all it is just a ScopedServiceTrackerMap with no other state and nothing specific to module.

Reply:
I can’t access the ScopedServiceTrackerMap from portal-impl. Can I?

Reply:
Ok it's possible. I'll do it.